analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/SymbolicExecution/CollectorTestCheck.cs

```diff
@@ -78,5 +78,22 @@ public void ValidateTag(string tag, Action<SymbolicValue> action)
 
         public void ValidateExitReachCount(int expected) =>
             exitReachedCount.Should().Be(expected);
+
+        public void ValidateExecutionCompleted() =>
+            executionCompletedCount.Should().Be(1);
+
+        public void ValidatePostProcess(int expected) =>
```

Comment:
This was not clear when reading the invocation
```suggestion
        public void ValidatePostProcessCount(int expected) =>
```